sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/PortSpec.java

```diff
@@ -30,10 +32,12 @@ public PortSpec(
             @JsonProperty("role") String role,
             @JsonProperty("principal") String principal,
             @JsonProperty("env-key") String envKey,
-            @JsonProperty("port-name") String portName) {
+            @JsonProperty("port-name") String portName,
+            @JsonProperty("network-names") Collection<String> networkNames) {
```

Comment:
- It looks like neither PortRequirement nor NamedVIPRequirement actually _use_ these names directly, there's just checks to see if they're empty or match a given format, in order to set boolean flags. So storing the full list may be more than what's needed.
- The network names also appear to vary on a per-task basis rather than a per-port basis, so this may be better placed in the `TaskSpec`